### PR TITLE
Feature/1841 1866 fixed length util

### PR DESCRIPTION
**Pull request by gamato**
Implements #1841 and #1866

### CODE REVIEW

File: qlib/FixedLengthUtil.qm [hunk 1]
```diff
@@ -537,6 +553,10 @@ my hash $h = $i.getValue();
                     # TODO: check here
                     break;
                 }
+                case "tab2space": {
+                    if (i.value.typeCode() != NT_INT || i.value < 0)
+                        throw errname, sprintf("expecting a non-negative int value to option %y; got %y (type %s) instead", i.key, i.value, i.value.type());
+                }
```
Comment (pvanek) [inline qlib/FixedLengthUtil.qm]: missing "break;"?

Comment (gamato) [reply qlib/FixedLengthUtil.qm]: Indeed!

**Review by davidnich — APPROVED** [review]
look good

**Comment by davidnich** [comment]
@pvanek please approve, and I'll make the merge

**Comment by pvanek** [comment]
It looks Fine for me.

**Pull request merged**